pages/overview.vue

```diff
@@ -125,10 +115,14 @@ export default class OverviewPage extends QiskitPage {
   }
 
   &__content-section-image {
-    position: relative;
-    right: -20%;
+    transform: translateX(20%);
     max-width: 100%;
     max-height: 30.5rem;
+
+    &__wrapper {
+      display: flex;
+      justify-content: flex-end;
+    }
```

Comment:
This changes were needed because of the nature of the images' lazy loading.

I needed to position the images this way so that they would continue to be placed with the right offset in the bigger viewports while still loading after the user scroll to them.